sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -17,6 +17,9 @@ public abstract partial class AsyncPageable<T> : System.Collections.Generic.IAsy
     }
     public static partial class AzureCoreExtensions
     {
+        public static dynamic ToDynamicFromJson(this System.BinaryData data) { throw null; }
+        public static dynamic ToDynamicFromJson(this System.BinaryData data, Azure.DynamicDataNameMapping propertyNameMapping) { throw null; }
```

Comment:
It's a bit strange that the type name is more generic than the parameter name, i.e. type name is about name mappings, but the parameter name is about specifically property mappings.